x/gov/legacy/proposals.go

```diff
@@ -0,0 +1,110 @@
+package gov
```

Comment:
It would be good to specify in either the file name or directory name _which_ migrations actually are taking place in this file aka (0_33_to_0_34) or something of the likes